spec/calendar.html

```diff
@@ -683,6 +716,46 @@ <h1>
         1. Return _merged_.
       </emu-alg>
     </emu-clause>
+
+    <emu-clause id="sec-temporal-toisodayofyear" type="abstract operation">
+      <h1>
+        ToISODayOfYear (
+          _year_: an integer,
+          _month_: an integer,
+          _day_: an integer,
+        ): an integer
+      </h1>
+      <dl class="header">
+        <dt>description</dt>
+        <dd>It implements the logic for the `Temporal.Calendar.prototype.dayOfYear` method, which is used for the ISO 8601 calendar.</dd>
+      </dl>
+      <emu-alg>
+        1. Let _epochDays_ be MakeDay(year, month - 1, day).
```

Comment:
I'd add "1. Assert: IsValidISODate(_year_, _month_, _day_) is **true**." to this operation and the other ones below, to express the additional constraint on year, month, and day that's not expressible in the types in the structured header.